src/GRA.Controllers/MissionControl/ProgramsController.cs

```diff
@@ -108,6 +108,10 @@ public async Task<IActionResult> Create(ProgramDetailViewModel model)
                 {
                     ModelState.AddModelError("Program.JoinBadgeName", "Please provide a name for the badge");
                 }
+                if (string.IsNullOrWhiteSpace(model.BadgeAltText))
+                {
+                    ModelState.AddModelError("BadgeAltText", "The Badge's Alt-Text is required.");
+                }
                 if (string.IsNullOrWhiteSpace(model.BadgeMakerImage) && model.BadgeUploadImage == null)
                 {
                     ModelState.AddModelError("BadgePath", "Please provide an image for the badge.");
```

Comment:
There should also be a ModelError check for if badge alt-text is filled in and there's no badge image.

Reply:
Fixed in commit af9d945713d8c860bbba206721d9861589a3135d